daemon/execdriver/windows/run.go

```diff
@@ -228,14 +172,23 @@ func (d *Driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	logrus.Debugf("CommandLine: %s", createProcessParms.CommandLine)
 
 	// Start the command running in the container.
-	var pid uint32
-	pid, err = hcsshim.CreateProcessInComputeSystem(c.ID, createProcessParms)
-
+	pid, stdin, stdout, stderr, err := hcsshim.CreateProcessInComputeSystem(c.ID, pipes.Stdin != nil, true, !c.ProcessConfig.Tty, createProcessParms)
```

Comment:
there's a lot of code duplication between Exec and Run, would it make sense to factor it out?